syntax = "proto3";

package hello;

message HelloRequest
{
    string message = 1;
}

message HelloResponse
{
    string message = 1;
}

service Greeter
{
    rpc SayHello(HelloRequest) returns(HelloResponse){}
}